generator/uscore/templates/access_verify_sequence.rb.erb

```diff
@@ -100,7 +108,7 @@ module Inferno
 
       def scope_granting_access(resource, scopes)
         scopes.split(' ').find do |scope| 
-          scope.start_with?("patient/#{resource}", 'patient/*') && scope.end_with?('*', 'read')
+          scope.start_with?("patient/#{resource}.", 'patient/*.') && scope.end_with?('.*', '.read')
```

Comment:
This would say `patient/Condition...blah...read` is ok. We should probably be more strict.
```
Regexp.new("patient\\/(\\*|#{resource})\\.(\\*|read)").match? scope
```
Or rather than putting the regex inline, define it in a method somewhere so we can unit test it.

Alternatively, we could just check all four options explicitly:
```
['patient/*.read', 'patient/*.*', "patient/#{resource}.read", "patient/#{resource}.*"].include? scope
```

Reply:
Good point.  I like the alternate form because it is more explicit to me.  I added tests for the `scope_granting_access` method to expose this issue and updated to your alternate form.